curriculum/challenges/english/25-front-end-development/lecture-the-var-keyword-and-hoisting/67329fbcfaf5ff5cdaa38a42.md

```diff
@@ -8,7 +8,41 @@ dashedName: what-is-the-var-keyword-and-why-is-it-no-longer-suggested-to-use-it
 
 # --description--
 
-Watch the video lecture and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What is the `var` keyword and why is it no longer suggested to use it?
+
+The `var` keyword in JavaScript is one of the original ways to declare variables. Its been part of the language since inception and for many years it remained the primary method for creating variables. However as JavaScript evolved and developers gained more experience with the language, certain drawbacks of using `var` became apparent leading to the introduction of `let` and `const` in 2015.
```

Comment:
```suggestion
The `var` keyword in JavaScript is one of the original ways to declare variables. It has been part of the language since inception and for many years it remained the primary method for creating variables. However as JavaScript evolved and developers gained more experience with the language, certain drawbacks of using `var` became apparent leading to the introduction of `let` and `const` in 2015.
```